gui/widgets/qopenglmousetracker.cpp

```diff
@@ -375,6 +383,9 @@ QColor QOpenGLMouseTracker::getBackgroundColor() const {
 }
 
 void QOpenGLMouseTracker::clear() {
+
+  SPDLOG_DEBUG("QOpenGLMouseTracker::clear() \"" + m_SubMeshes->name + "\"");
```

Comment:
This debug line may segfault if m_SubMeshes is a nullptr

Reply:
asan ci job catches this: https://github.com/spatial-model-editor/spatial-model-editor/actions/runs/9076985488/job/24940866694?pr=986

Reply:
I was using it only for debugging purposes. I was trying to catch the usual destructor calls. It wasn't meant to remain inside the code, thanks.